### ISSUE TITLE
New prey and imbuement system

### ISSUE THREAD

**Issue by westwol** [malucooo/Otxserver-New#141]
I saw in TibiaKing that the server has included Prey and Imbuement System and just tested the datapack and it seems to be missing or at least haven't been merged into master.

Thanks in advance.

**Comment by jlcvp** [comment]
Those are WIP features. Will be implemented soon. If you have the availability and the knowledge, feel free to open a merge request using the otland post of those system as a base.

Thanks in advance,

Leu

**Comment by newacc12** [comment]
We should probably describe that in the TibiaKing post or on the main page here. I guess a lot of people will ask the same question.

**Comment by westwol** [comment]
@jlcvp I have some advances related to these systems but still missing features what I want to know is what have you guys already achieved? As you may know the systems recently released in TibiaKing are not fully working at all. I've been testing out these systems lately and found out these info below:

Prey System: 

Missing features.
- Applying the boosts: block damage, boost damage, loot drop chance and xp boost to the player.
- preyStamin usage.

Imbuement System: 

Missing features.
- Attributes such as: magic, sword, axe, distance, etc. (IMO they should be handled either by an onMovement script or modifyng the source code)
- Items and money removal once the imbue is done.
- imbuementStamina usage.

If I am missing something please feel free to post it.

Westwol.

**Comment by mattyx14** [comment]
Great news
